src/client/components/Molecules/FooterIcons/FooterIcons.module.css

```diff
@@ -0,0 +1,8 @@
+.FooterIcons {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  gap: 14px;
```

Comment:
What do you think about creating global `variable` for centering any kind of html just for other tags for future ❓ 
It will shorten a bit our css code 💡

Reply:
We could reuse it in many cases 🤞

Reply:
Good idea, but I think that we should have done that at the beginning of the project :P
At this point it may cause weird styles in different icons.

Reply:
Its okey without it anyway 👍 
Just thinking on clean code 🙌